main.tf

```
@@ -457,9 +465,15 @@ resource "aws_ecs_service" "ignore_changes_task_definition" {
     }
   }
 
+  triggers = local.redeployment_trigger
```

How about renaming this to triggers instead, should there ever be other triggers?
```suggestion
  triggers = local.triggers
```

---

Again I would leave it like that because we keep the repo as close as possible from the fork